kvmagent/kvmagent/test/test_prometheus_plugin.py

```
@@ -0,0 +1,253 @@
+
+for _p in (_kvmagent_root, _zstacklib_root):
+    if _p not in sys.path:
+        sys.path.insert(0, _p)
+
+# ---------------------------------------------------------------------------
+# 2. Stubs for heavy / unavailable dependencies BEFORE the import
+# ---------------------------------------------------------------------------
+
+# log
+_mock_log = types.ModuleType('zstacklib.utils.log')
+_mock_log.get_logger = lambda name: mock.MagicMock()
```

Ruff (0.15.5)

[warning] Unused lambda argument: `name`

(ARG005)